app/src/main/java/com/kccreate/intelligentlockdown/extensions/FragmentExtensions.kt

```diff
@@ -0,0 +1,21 @@
+package com.kccreate.intelligentlockdown.extensions
+
+import android.view.View
+import androidx.annotation.IdRes
+import androidx.core.os.bundleOf
+import androidx.fragment.app.Fragment
+
+
+inline fun <reified T : View> Fragment.bindView(@IdRes id: Int): Lazy<T> = lazy {
```

Comment:
we could just use view bindings

Reply:
100% true, never tried to use them though, so I didn't want to spend time on learning how to use them.
Are you happy to replace this in a future PR?